docs/tutorials/data/hess.ipynb

```diff
@@ -12,7 +12,17 @@
     "\n",
```

Comment:
<p>unnecessary</p><p> </p><p>Do we really want to put the Note written like this? After all we should have the opportunity to introduce data at the DL4 level.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3350/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>